src/components/ColorPicker.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+
+<div class="color-picker-overlay" @click="closeColorPicker">
+  <div class="container" :style="containerPosition" ref="container" @click.stop>
+  <div @click="saveColor" class="button save">&check;</div>
+  <v-color-picker 
+  class="myColorPicker"
+  v-on:input = "colorPickerColor"
+  dot-size = "40"
+  mode="hexa"
+  v-bind:dark= "true"
+  v-bind:value = "oldColor"
+  width="250"
+  canvas-height="100"
+></v-color-picker>
+<div @click="closeColorPicker" class="button close">&#9932;</div>
```

Comment:
Formatting is off with this section. To format the document in VSCode use Shift + Alt + F